backend/app/api/admin_routes/stats.py

```diff
@@ -31,3 +37,37 @@ def chat_origin_trend(
 ) -> ChatStats:
     stats = chat_repo.chat_trend_by_origin(session, start_date, end_date)
     return ChatStats(start_date=start_date, end_date=end_date, values=stats)
+
+class ChatOrigin(UUIDBaseModel):
+    origin: str
+
+@router.get("/admin/stats/chats/origins")
```

Comment:
For the route path naming, you can share your points on the two naming methods `/admin/stats/chats/origins` vs `/admin/chats/origins`